src/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ option(server "Build the server (Murmur)" ON)
 
 option(qssldiffiehellmanparameters "Build support for custom Diffie-Hellman parameters." ON)
 
-option(bonjour "Build support for Bonjour." ON)
+option(zeroconf "Build support for zeroconf (mDNS/DNS-SD)." ON)
```

Comment:
1. In your commit message you state that the option is still called `bonjour` so that seems to be outdated/incorrect
2. Isn't this only needed for Windows systems? In that case we should only default-enable it on Windows

Reply:
1. I'm referring to the server option, the one in the configuration file.
2. mDNS/DNS-SD is used on all platforms. The only difference is that now on Windows we use the native API, if available.

Reply:
Ah okay I see